spec/controllers/tenants_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe TenantsController, type: :controller do
```

Comment:
always write specs!!!
Code not covered with specs won't be merged.

Reply:
remove this file